Makefile

```diff
@@ -122,6 +122,7 @@ EKS_A_RELEASE_CROSS_PLATFORMS := $(foreach platform,$(EKS_A_PLATFORMS),eks-a-rel
 
 DOCKER_E2E_TEST := TestDockerKubernetes121SimpleFlow
 LOCAL_E2E_TESTS ?= $(DOCKER_E2E_TEST)
+CLOUDSTACK_E2E_TEST := TestCloudStackKubernetes121SimpleFlow
```

Comment:
We don't really need these Makefile changes

Reply:
Are you suggesting to remove all the new targets and just run the test tool manually?